test/integration/helpers/inputs.js

```diff
@@ -1,5 +1,15 @@
 'use strict';
-/*jshint esversion: 6 */
+
+
+const setInputValue = (selector, value) => {
+  var input = element(selector);
+  input.clear();
+  input.sendKeys(value);
+  // do we want the expect here? it is helpful
```

Comment:
We either do or we don't :) 

I don't have a strong opinion, but I'd rather just decide then ask a question in a comment. I think it's fine either way.